app/lib/promoted-projects.js

```diff
@@ -0,0 +1,42 @@
+const PROMOTED_PROJECTS = {
+  production: {
+    60: {
+      image: '/assets/featured-projects/60.jpg',
+      title: 'Chimp & See'
+    },
+    21: {
+      image: '/assets/featured-projects/21.jpg',
+      title: 'Galaxy Zoo'
+    },
+    376: {
+      image: '/assets/featured-projects/376.jpg',
+      title: 'Shakespeare\'s World'
+    },
+    3098: {
+      image: '/assets/featured-projects/26.jpg',
+      title: 'Muon Hunter'
+    }
+  },
+  other: {
+    1731: {
+      image: '/assets/featured-projects/60.jpg',
+      title: 'Chimp & See'
+    },
+    1732: {
+      image: '/assets/featured-projects/21.jpg',
+      title: 'Galaxy Zoo'
+    },
+    1733: {
+      image: '/assets/featured-projects/376.jpg',
+      title: 'Shakespeare\'s World'
+    },
+    1734: {
+      image: '/assets/featured-projects/3098.jpg',
+      title: 'Muon Hunter'
+    }
+  }
+};
+
+const env = process.env.NODE_ENV === 'production' ? 'production' : 'other';
```

Comment:
The app is unable to load when I use the `?env=production` query at the end of the staging url, so maybe some sort of `envFromBrowser = locationMatch(/\W?env=(\w+)/);` variable is in order?

Reply:
I think it should work if you `npm start NODE_ENV=production`, but I'm not sure what this is supposed to do tbh. Do you remember what was your intention here?

Reply:
Yes, that does work. I just know that sometimes when testing staging branches or local work I will apply the `?env=production` ending to the url to see how things appear in production. In this case I did https://reinstate-promoted-projects.pfe-preview.zooniverse.org/?env=production and saw the loading spinner in place of promoted projects because the client is trying to fetch project ids for promoted projects in staging (1731, 1732, 1733, 1734) while pinging the production server.

Reply:
I see what you, but why is that line need at all?

Reply:
I was thinking the locationMatch line would be used similarly to how it is used in the [split-config](https://github.com/zooniverse/Panoptes-Front-End/blob/595b686eb1a244e924f3df3ff317bba46f3fc551/app/lib/split-config.js#L5) or [pusher-env](https://github.com/zooniverse/Panoptes-Front-End/blob/595b686eb1a244e924f3df3ff317bba46f3fc551/app/lib/pusher-env.js)

Reply:
I've added it in.